src/pages-and-resources/teams/GroupEditor.jsx

```diff
@@ -31,6 +32,11 @@ const GroupEditor = ({
   const [isOpen, setOpen] = useState(group.id === null);
   const initiateDeletion = () => setDeleting(true);
   const cancelDeletion = () => setDeleting(false);
+  // This is a temporary workaround to disable the OPEN MANAGED team type until it is fully adopted
+  // For more information, see: https://openedx.atlassian.net/wiki/spaces/COMM/pages/3885760525/Open+Managed+Group+Type
+  const isGroupTypeEnabled = (groupType) => (
```

Comment:
I can't find tests for this component. Could someone guide me in the right direction? Thanks!